orangecontrib/spectroscopy/io/util.py

```diff
@@ -99,15 +99,13 @@ def read_tile(self):
 
     def read(self):
```

Comment:
@stuart-cls, could you please verify if my changes did not break anything? I did not explicitly test them. They do pass the tests though. Thanks!

Reply:
I tested with the included workflow on both the example dataset and a larger mosaic. It works fine.

Reply:
Thanks!